packages/components/Button/src/Button.tsx

```diff
@@ -83,9 +83,15 @@ export const Button = compose<ButtonType>({
           {loading && <ActivityIndicator />}
           {shouldShowIcon && iconPosition === 'before' && <Slots.icon {...iconProps} accessible={false} />}
           {React.Children.map(children, (child) =>
-            typeof child === 'string' ? <Slots.content key="content">{child}</Slots.content> : child,
+            typeof child === 'string' ? (
+              <Slots.content accessible={false} key="content">
```

Comment:
It would make sense to do a pass over the compound button and the FAB to make sure they don't need to be updated.

Reply:
ToggleButton too?

Reply:
Done for all.